packages/bff/src/config.ts

```diff
@@ -42,18 +41,10 @@ const config = {
   postgresql: {
     connectionString: env.DB_CONNECTION_STRING,
   },
-  cookieName: env.COOKIE_NAME,
   secret: env.SESSION_SECRET,
   enableHttps: env.ENABLE_HTTPS,
   cookieMaxAge: env.COOKIE_MAX_AGE,
+  redis: env.REDIS_CONNECTION_STRING,
```

Comment:
Valg navngivning, `redis` og aligner ikke helt med øvrig navngivning. Hvorfor ikke bare `redisConnectionString`?  ev vurdere nivå 2 som med `postgresql`.

Reply:
Stemmer for nivå 2. Så er det mulig å ekspandere med andre ting som "enabled" osv. Logikken blir veldig encapsulated